### DISCUSSION TITLE
The value of the AUTO_INCREMENT field increases too quickly

### DISCUSSION THREAD

**Discussion by eleven-f** [apache/doris#36851] [RFC]
The experiment found that there were only about 26 million data in the table, but the IDs automatically grew to 136 million. Will this growth rate cause the IDs of a large table (expected to store about 10 billion data) to quickly grow to the upper limit of bigint? Can it support long-term data insertion?
![image](https://github.com/apache/doris/assets/30048177/a4cb15be-d56a-4560-ab2f-d0830a42181a)
![image](https://github.com/apache/doris/assets/30048177/08ec7aca-7537-49ca-90f3-e89fc10bbb5f)

**Comment by ixzc** [comment]
it can support long-term data insertion.
